dev-proxy/devproxyrc.json

```diff
@@ -2,198 +2,22 @@
     },
     {
-      "name": "CachingGuidancePlugin",
-      "enabled": true,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll",
-      "configSection": "cachingGuidance"
-    },
-    {
-      "name": "RateLimitingPlugin",
-      "enabled": false,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll",
-      "configSection": "rateLimiting"
-    },
-    {
-      "name": "MockResponsePlugin",
-      "enabled": false,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll",
-      "configSection": "mocksPlugin"
-    },
-    {
-      "name": "GraphMockResponsePlugin",
-      "enabled": true,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll",
-      "configSection": "mocksPlugin"
-    },
-    {
-      "name": "GraphRandomErrorPlugin",
-      "enabled": true,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll",
-      "configSection": "graphRandomErrorsPlugin"
-    },
-    {
-      "name": "ExecutionSummaryPlugin",
-      "enabled": false,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll",
-      "configSection": "executionSummaryPlugin"
-    },
-    {
-      "name": "MinimalPermissionsPlugin",
+      "name": "RetryAfterPlugin",
       "enabled": true,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll",
-      "configSection": "minimalPermissionsPlugin"
-    },
-    {
-      "name": "MinimalPermissionsGuidancePlugin",
-      "enabled": false,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll"
-    },
-    {
-      "name": "MockGeneratorPlugin",
-      "enabled": false,
-      "pluginPath": "plugins\\dev-proxy-plugins.dll"
-    },
-    {
-      "name": "OpenApiDocGeneratorPlugin",
-      "enabled": false,
       "pluginPath": "plugins\\dev-proxy-plugins.dll"
     }
   ],
   "urlsToWatch": [
-    "https://graph.microsoft.com/v1.0/*",
-    "https://graph.microsoft.com/beta/*",
-    "https://graph.microsoft.us/v1.0/*",
-    "https://graph.microsoft.us/beta/*",
-    "https://dod-graph.microsoft.us/v1.0/*",
-    "https://dod-graph.microsoft.us/beta/*",
-    "https://microsoftgraph.chinacloudapi.cn/v1.0/*",
-    "https://microsoftgraph.chinacloudapi.cn/beta/*",
-    "https://*.sharepoint.*/*_api/*",
-    "https://*.sharepoint.*/*_vti_bin/*",
-    "https://*.sharepoint-df.*/*_api/*",
-    "https://*.sharepoint-df.*/*_vti_bin/*"
+    "!https://jsonplaceholder.typicode.com/posts",
+    "https://jsonplaceholder.typicode.com/*"
   ],
-  "mocksPlugin": {
-    "mocksFile": "responses.json"
-  },
-  "graphRandomErrorsPlugin": {
-    "allowedErrors": [ 429, 500, 502, 503, 504, 507 ]
-  },
-  "executionSummaryPlugin": {
-    "groupBy": "url"
-  },
-  "minimalPermissionsPlugin": {
-    "type": "delegated"
-  },
-  "cachingGuidance": {
-    "cacheThresholdSeconds": 5
-  },
-  "latencyPlugin": {
-    "minMs": 200,
-    "maxMs": 10000
-  },
-  "devTools": {
-    "preferredBrowser": "Edge"
-  },
-  "rateLimiting": {
-    "costPerRequest": 2,
-    "rateLimit": 120,
-    "retryAfterSeconds": 5
-  },
-  "rate": 50,
```

Comment:
Don't we want to keep the global settings for easy change?

Reply:
Good catch, I'll add `rate` and also `labelMode` and `logLevel`.